/xavier_ssd/TrekBot/TrekBot2_WS/build/razor_imu_9dof/setup_custom_pythonpath.sh: 5: exec: /xavier_ssd/TrekBot/TrekBot2_WS/src/razor_imu_9dof/cfg/imu.cfg: Permission denied
make[2]: *** [/xavier_ssd/TrekBot/TrekBot2_WS/devel/.private/razor_imu_9dof/include/razor_imu_9dof/imuConfig.h] Error 126
make[1]: *** [CMakeFiles/razor_imu_9dof_gencfg.dir/all] Error 2
make: *** [all] Error 2
